src/theVoice/index.tsx

```diff
@@ -1,22 +1,15 @@
 import styled from 'styled-components'
-import { space, color, font, responsiveBreakpoints } from '_utils/branding'
+import { space, color, responsiveBreakpoints } from '_utils/branding'
 
 import TheVoice from './TheVoice'
 
 const StyledTheVoice = styled(TheVoice)`
-  & {
-    text-align: center;
-    font-size: ${font.xl.size};
-    margin: ${space.xxl} 0;
-  }
+  margin: 0;
+  padding: ${space.xl} 0 calc(${space.xl} + ${space.m}); /* 32px it's not on scale but on spec :() */
```

Comment:
I'd rather add `32px` to our scale than doing some calculation in CSS.
If you don't want to make a too big change in the codebase by doing that, you can add a scoped variable in theVoice. e.g. `const paddingBottom = '32px'`